services/movie_session.py

```diff
@@ -0,0 +1,55 @@
+from django.utils.datetime_safe import datetime
+from db.models import MovieSession, Movie, CinemaHall
+from django.utils.dateparse import parse_date
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> MovieSession:
+    movie = Movie.objects.get(id=movie_id)
+    cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie=movie,
+        cinema_hall=cinema_hall
+    )
+
+
+def get_movies_sessions(session_date: datetime = None) -> MovieSession:
+    if session_date:
+        return MovieSession.objects.filter(
+            show_time__date=parse_date(session_date)

```

Comment:
The `session_date` parameter should be a string to match the `parse_date` function usage. Consider changing the parameter type to `str`.